WordPress/Jetpack/Classes/NUX/JetpackPrologueViewController.swift

```diff
@@ -12,6 +13,12 @@ class JetpackPrologueViewController: UIViewController {
         return view
     }()
 
+    private lazy var motion: CMMotionManager = {
```

Comment:
Apple recommends a singleton for `CMMotionManager` ([Source](https://developer.apple.com/documentation/coremotion/cmmotionmanager)):


> Create only one [CMMotionManager](https://developer.apple.com/documentation/coremotion/cmmotionmanager) object for your app. Multiple instances of this class can affect the rate at which data is received from the accelerometer and gyroscope.

So we may have an issue with this and the [one used here for the About screen](https://github.com/Automattic/AutomatticAbout-Swift/blob/d3e8b4d8133984d9ae20936a55e747d27661e2a1/Sources/AutomatticAbout/AutomatticAppLogosCell.swift#L114).

Thoughts @Gio2018 and @salimbraksa?